nessie/src/main/java/org/apache/iceberg/nessie/NessieUtil.java

```diff
@@ -82,10 +92,26 @@ static ContentsKey toKey(TableIdentifier tableIdentifier) {
     return key;
   }
 
+  static CommitMeta buildCommitMetadata(String commitMsg, Map<String, String> catalogOptions) {
+    Preconditions.checkArgument(null != catalogOptions, "catalogOptions must not be null");
+    ImmutableCommitMeta.Builder cm = CommitMeta.builder().message(commitMsg)
+        .author(NessieUtil.getCommitAuthor(catalogOptions));
+    cm.putProperties(APPLICATION_TYPE, "iceberg");
```

Comment:
Looks like this changes the property keys that are passed to Nessie. Is that okay?

Reply:
yep that change was on purpose as I wanted those properties to be named the same way (using `-` vs `.`)